packages/main/src/ToggleButton.ts

```diff
@@ -44,7 +44,12 @@ class ToggleButton extends Button {
 	@property({ type: Boolean })
 	pressed!: boolean;
 
-	_onclick() {
+	_onclick(e: MouseEvent) {
+		if (this.disabled) {
+			e.preventDefault();
+			e.stopPropagation();
```

Comment:
Do we have to return here, because on the next line the pressed state will be inverted `this.pressed = !this.pressed;`, or it's ok to be inverted.